phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java

```diff
@@ -940,4 +941,70 @@ public void testOffsetExplain() throws SQLException {
             assertTrue(explainStringBuilder.toString().contains("With RVC Offset"));
         }
     }
+
+    @Test public void testGlobalIndexViewAccess() throws Exception {
+        String
+                ddl =
```

Comment:
nit: weird line break

`String ddl =` should be a single line?

Reply:
i think i used the phoenix code formatter and it did this i can see if i can trick it into formatting nicer.